miso-web/src/main/webapp/scripts/hot_qc.js

```diff
@@ -134,7 +134,9 @@ HotTarget.qc = function(qcTarget) {
         pack: function(qc, flat, errorHandler) {
           // Do nothing; this never goes to the server
         }
-      }];
+      }, HotUtils.makeColumnForText('Note', true, 'description', {
```

Comment:
Please label column 'Description' for consistency with other MISO objects. Same in `list_qc.js`